src/browser/components/molecules/TaskList/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react'
+import Layout from '../../molecules/Layout/index'
+import Task from '../Task/index'
+import styles from './index.css';
+const moment = require('moment')
+
+export default class TaskList extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      tasks: []
+    }
+  }
+
+  componentWillMount(){
```

Comment:
If you are changing state, then you should use `componentDidMount` instead of `componentWillMount`. `componentWillMount` does not re-render the page if the state changes. More info found in the docs - https://facebook.github.io/react/docs/react-component.html